docs/getting-started/index.md

```diff
@@ -0,0 +1,522 @@
+---
+order: 1
+parent:
+  order: 1
+  title: Getting Started with Starport
+---
+
+# Getting Started
+
+## What is Starport?
+
+Starport is an easy to use CLI tool for creating sovereign blockchains with Cosmos SDK. Cosmos SDK is the world's most popular framework for building blockchains. Both Starport CLI and Cosmos SDK are written in the Go programming language.
+
+## Starting a New Starport Project
+
+### Installing Go
+
+Before you start using Starport, you should check that your system has Go installed. To do so run:
+
+```
+go version
+```
+
+If you see `go1.16` (or higher), then you have the right version of Go installed. If the output is `command not found` or installed version of Go is older than 1.16, [install or upgrade Go](https://golang.org/doc/install).
+
+### Installing Starport CLI
+
+To install Starport run the following command:
+
+```
+curl https://get.starport.network/starport! | bash
+```
+
+This command will fetch the `starport` binary and install it into `/usr/local/bin`. If this command throws a permission error, lose `!` and it will download the binary in the current directory, you can then move it manually into your `$PATH`.
+
+### Creating a Blog-Chain
+
+Starport comes with a number of scaffolding commands that are designed to make development easier by creating everything that's necessary to start working on a particular task. One of these tasks is a `scaffold chain` command which provides you with a foundation of a fresh Cosmos SDK blockchain so that you don't have to write it yourself.
+
+To use this command, open a terminal, navigate to a directory where you have permissions to create files, and run:
+
+```
+starport scaffold chain blog
+```
+
```

Comment:
The scaffolded blockchain is a ready-to-use blockchain. It has basic modules installed, similar as you might know from the [Cosmos Hub](https://hub.cosmos.network/main/hub-overview/overview.html#). 
Adding your own logic and transaction types is explained in the coming chapters.

Reply:
Also used to the current syntax: `starport app github.com/alice/chain` (I was thinking this would be released after we release a new version with restructured commands, but decided not to wait)